### PR TITLE
DO NOT MERGE - Change vagov_info_token to JSON

### PR DESCRIPTION
**Pull request by rileyanderson**
## Summary
Value is currently set as a stringified Ruby hash which is unnecessarily complex to parse. 
- Change value to JSON  

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#55351
- department-of-veterans-affairs/va.gov-team#55344

## Testing done
- Updated specs
- Tested login, cookie value was json

## Screenshots
Screen Shot 2023-03-17 at 1 33 35 PM

## What areas of the site does it impact?
Sessions

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by asg5704 — APPROVED** [review]
Tested locally, works as expected. Still need to wait on Afia's PR before merging.

**Comment by asg5704** [comment]
DON'T MERGE YET!

**Comment by rileyanderson** [comment]
> DON'T MERGE YET!

taunting

**Pull request merged**
